starfish/pipeline/features/spots/detector/_base.py

```diff
@@ -1,12 +1,8 @@
-from typing import Tuple
-
 from starfish.image import ImageStack
 from starfish.pipeline.algorithmbase import AlgorithmBase
-from starfish.pipeline.features.spot_attributes import SpotAttributes
-from starfish.pipeline.features.encoded_spots import EncodedSpots
 
 
 class SpotFinderAlgorithmBase(AlgorithmBase):
-    def find(self, hybridization_image: ImageStack) -> Tuple[SpotAttributes, EncodedSpots]:
+    def find(self, hybridization_image: ImageStack):
```

Comment:
Is this supposed to return IntensityTable once LocalMaxPeakFinder is updated?

Reply:
Yep, that's right.

Reply:
Pls TODO: this so we don't forget.